datamap/mv/core/src/test/scala/org/apache/carbondata/mv/rewrite/TestAllOperationsOnMV.scala

```diff
@@ -614,6 +615,16 @@ class TestAllOperationsOnMV extends QueryTest with BeforeAndAfterEach {
     assert(!cacheAfterDrop.asScala.exists(key => key.startsWith(mvPath)))
   }
 
+  test("test case insensitive create & drop mv command") {
+    sql("drop table IF EXISTS mv_case_insensitive")
+    sql("create table mv_Case_InSenSitive(name string, age int) stored by 'carbondata'")
+    sql("insert into mv_case_insensitive select 'aa',20")
+    sql("insert into mv_Case_InSenSitive select 'aa',20")
+    sql("drop datamap if exists dm_InSenSitive ")
+    sql("create datamap dm_Insensitive using 'mv' as select name from mv_case_insensitive group by name")
+    sql("drop datamap dm_InSenSitive ")
```

Comment:
Execute select queries and verify with case_insensitive table name

Reply:
@Indhumathi27 Done.